style.css

```diff
@@ -355,5 +381,136 @@ h1 span {
   color: #42526e;
   text-align: center;
   padding: 5px;
-  margin: 5px 0 200px 0;
+  margin: 5px 0 10px 0;
+}
+
+hr {
+  display: none;
+}
+
+@media screen and (min-width: 992px) {
+  header {
+    justify-content: center;
+  }
+
+  .desk-nav {
+    align-items: center;
+    list-style: none;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    background: transparent;
+    position: static;
+  }
+
+  .desk-nav li {
+    margin: 0 10px;
+    font-weight: bold;
+  }
+
+  .logo {
+    display: none;
+  }
+
+  nav .fa-bars {
+    display: none;
+  }
+
+  .project-container {
+    grid-template-columns: repeat(3, 1fr);
+  }
```

Comment:
- [ ] [OPTIONAL] Everything is working but there is too much space between card elements. Please kindly fix this issue by adding `justify-content: center`. If you don't use this, your design may break in bigger desktops screens. You can see the difference between your and Figma design 🔽

## Your design
![image](https://user-images.githubusercontent.com/69011963/144591715-d6459dbe-b4b0-4bc4-9dab-fa623d9c2bba.png) 

## Figma design
 ![image](https://user-images.githubusercontent.com/69011963/144591868-c317bd58-7d93-4078-b93c-c646db1869b7.png)